memgpt/constants.py

```diff
@@ -16,8 +16,18 @@
 INITIAL_BOOT_MESSAGE_SEND_MESSAGE_FIRST_MSG = STARTUP_QUOTES[2]
 
 # Constants to do with summarization / conversation length window
-MESSAGE_SUMMARY_WARNING_TOKENS = 7000  # the number of tokens consumed in a call before a system warning goes to the agent
+# The max amount of tokens supported by the underlying model (eg 8k for gpt-4 and Mistral 7B)
+LLM_MAX_TOKENS = 8000  # change this depending on your model

```

Comment:
@vivi you can test that this works by intentionally lowering LLM_MAX_TOKENS + lowering it on the web UI backend